Chore: Upgrade all Dependencies and resolve any breaking changes (#16)
natemoo-re/astro-remote: packages/astro-remote/README.md

<Markdown content={content} sanitize={{ allowComponents: true }} components={{ MyCustomComponent }} />
```

### Using Marked Extensions

> If you'd like to use custom Marked Extensions it is now as easy as doing the following:

Reviewer: ```suggestion
If you'd like to extend the underlying [Marked](https://marked.js.org/using_pro) behavior, the `marked` prop accepts `extensions`.
```

```astro
---
import { Markdown } from 'astro-remote';

> import MyCustomComponent from '../components/MyCustomComponent.astro';

Reviewer: ```suggestion
```

import markedAlert from 'marked-alert'

const content = await fetch('http://my-site.com/api/v1/post').then(res => res.text());
---

> <Markdown content={content} sanitize={{ allowComponents: true }} components={{ MyCustomComponent }} marked={{ extensions: [ markedAlert() ] }} />

Reviewer: ```suggestion
<Markdown content={content} marked={{ extensions: [ markedAlert() ] }} />
```
